### DISCUSSION TITLE
Cant verify university ID Proof for GitHub student pack

### DISCUSSION THREAD

**Discussion by Shayaan-Fayaz** [community/community#77166] [GitHub Education] [answered]
### Hi everyone,

I Cant verify university ID Proof for GitHub student pack, please help

![image](https://github.com/community/community/assets/110037144/bdb328ed-badd-4d29-84b2-aeed20a9e39b)

**Comment by CheetahCodes21** [comment]
Verifying your university ID proof for the GitHub Student Developer Pack can sometimes be challenging due to various factors, such as the type of ID, the format of the document, or the way you upload it. Here are some troubleshooting steps you can try to resolve the issue:

1. **Ensure your ID is valid and verifiable:** Check that your university ID is still valid and has not expired. Make sure the ID includes your full name, your university name, and your current enrollment status.

2. **Use a clear and complete image:** Take a clear and well-lit photo of your university ID. Ensure the entire ID is visible and all details are legible. Avoid using cropped or blurry images.

3. **Upload the correct file format:** GitHub supports various file formats for uploading ID proofs. Try uploading your ID as a JPEG (.jpg), PNG (.png), or PDF (.pdf) file.

4. **Check the file size limit:** The maximum file size for uploading ID proofs is 5MB. If your ID image exceeds this limit, try resizing or compressing it to a smaller size.

5. **Double-check the upload process:** Make sure you're uploading the correct image file and selecting the appropriate option for uploading your ID proof. Follow the on-screen instructions carefully.

6. **Clear your browser cache and cookies:** Sometimes, temporary data stored in your browser can interfere with the upload process. Try clearing your browser cache and cookies and then uploading your ID again.

7. **Try a different browser:** If the issue persists, try using a different web browser to upload your ID. Sometimes, browser compatibility issues can cause problems.

8. **Contact GitHub Support:** If you've tried all these steps and are still unable to verify your ID, reach out to GitHub Support for further assistance. They can help troubleshoot the issue and provide personalized guidance.

Remember to be patient and persistent throughout the verification process. GitHub Support may need to review your ID manually, which can take some time.

**Reply by seliminsan** [reply]
There is no option to upload files, only the option to take photos.

**Reply by CheetahCodes21** [reply, marked as answer]
what I had done was kept my ID in front of the web cam and take the photo

**Reply by Shayaan-Fayaz** [reply]
Thank you for the help. It now got submitted
